cli/azd/pkg/apphost/testdata/TestAspireDockerGeneration-nodeapp.snap

```diff
@@ -14,7 +14,7 @@ properties:
         autoConfigureDataProtection: true
     ingress:
       external: true
-      targetPort: {{ targetPortOrDefault 3000 }}
+      targetPort: {{ targetPortOrDefault -3000 }}
```

Comment:
Ideally, as a reader, I think setting any explicit value means "use this value". If unset or 0, it means derive from port from the `dotnet publish` operation. I'm thinking the negative integer to indicate "unused" may be a little unintuitive. 

Is there a way we could update the generation to allow for this? Perhaps we're trying to avoid some special case here around port handling?

Reply:
updated to handle this during generation